assets/src/apps/delivery/store/features/groups/actions/deck.ts

```diff
@@ -65,7 +65,7 @@ export const initializeActivity = createAsyncThunk(
     const visitOperation: ApplyStateOperation = {
       target: `session.visits.${currentSequenceId}`,
       operator: '+',
-      value: 1,
+      value: 0,
```

Comment:
hmm if we really aren't supposed to do this here, then I think we should just remove it all together. no point in *adding* zero